aggregator/src/aggregator/aggregation_job_creator.rs

```diff
@@ -347,6 +380,164 @@ impl<C: Clock + 'static> AggregationJobCreator<C> {
             .await?)
     }
 
+    #[tracing::instrument(skip(self), err)]
+    async fn create_aggregation_jobs_for_fixed_size_task_no_param<
+        const L: usize,
+        A: vdaf::Aggregator<L, AggregationParam = ()>,
+    >(
+        self: Arc<Self>,
+        task: Arc<Task>,
+        max_batch_size: usize,
+    ) -> anyhow::Result<()>
+    where
+        for<'a> &'a A::AggregateShare: Into<Vec<u8>>,
+        A::PrepareMessage: Send + Sync,
+        A::PrepareState: Send + Sync + Encode,
+        A::OutputShare: Send + Sync,
+        for<'a> &'a A::OutputShare: Into<Vec<u8>>,
+    {
+        Ok(self
```

Comment:
nit: if you did
```
let ret = self.datastore.run_tx(...);
Ok(ret)
```
...then you could de-indent the bulk of the method by one tab, which maybe gives `cargo fmt` a little more breathing room to lay out the code, but it's no big deal.

Reply:
I tried this out, but unfortunately `rustfmt` didn't actually de-indent anything -- all the lines after the edited line are indented equivalently.

(Since the `Ok` result will always be `()`, I tried returning `Ok(())` instead of assigning to `ret`, but unfortunately the results were the same.)